lib/copydir.c

```diff
@@ -400,6 +400,7 @@ static int copy_entry (const struct path_info *src, const struct path_info *dst,
 {
 	int err = 0;
 	struct stat sb;
+	struct stat tmp_sb;
```

Comment:
tmp_ is a bit unclear of a name.  The other one is also temporary.

Maybe dst_sb?

Reply:
The other one (`sb`) is not temporary; its content is read and used.   This one is really temporary and just used to satisfy the `fstatat()` API.

Reply:
Ahh, I see.  LGTM, then.